### DISCUSSION TITLE
Are there any time estimates for "eventual" eventually consistency level?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [milvus-io/milvus#18898] [Q&A and General discussion] [answered]
Are there any time estimates for "eventual" eventually consistency level? 
for instance: I add a lot of new data, when this data will be visible?
In my test (for face recognition) I add a new face and it is immediately visible.

Are there any time estimates? Or does it flushes after 10th or 100th new entry?

Does it affect addition? Or only deletion? 

Have you done any tests?

**Comment by yhmo** [comment, marked as answer]
Milvus checks data consistency with an interval(200ms).
- So, if you use "Strong", the search request will wait 200ms ~ 400ms for the next checkpoint.
- If you use "Eventually", the search request will be executed instantly, but the data of the next checkpoint is invisible for this request. The invisible time gap might be 200ms ~ 400ms.
